pii_recognizer/pii_recognizer.py

```diff
@@ -44,35 +46,42 @@
 class PatternRecognizerFactory:
     """
     Factory for creating pattern recognizers, it can be extended in the future to
-    add more regex pattern for different entities.
+    add more regex pattern for different entities. For the pattern recognizer to work,
+    we need construct a list of regex patterns for each entity.
     """
 
-    # create a list of pattern recognizers
+    _DEFAULT_ENTITIES = {
+        "CREDIT_CARD": [pa.Pattern("CREDIT_CARD", r"\b(?:\d[ -]*?){13,16}\b", 0.5)],
+        "SSN": [pa.Pattern("SSN", r"\b\d{3}-?\d{2}-?\d{4}\b", 0.5)],
+        "PHONE": [pa.Pattern("PHONE", r"\(?\d{3}\)?[-.\s]?\d{3}[-.\s]?\d{4}", 0.5)],
+        "EMAIL": [pa.Pattern("EMAIL", r"\S+@\S+", 0.5)],
+    }
 
-    @staticmethod
-    def create_pattern_recognizer():
+    # create a list of pattern recognizers
+    @classmethod
+    def _create_pattern_recognizer(cls):
```

Comment:
I think it should be non-protected as you use it in lines 486 and 495

```suggestion
    def create_pattern_recognizer(cls):
```

Reply:
If it appear non-protected it will be counted as an handler, so it must remain unprotected